### ISSUE TITLE
Fix some import errors in twisted/spread on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8621]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8621|
|Type|enhancement|
|Created|2016-07-18 08:21:42Z|
|Branch|https://github.com/twisted/twisted/tree/8621-rodrigc-spread|

When trying to run test_jelly under Python 3,
there are some import errors in twisted/spread/flavors.py:

```
  File "/Users/crodrigues/twisted7/twisted/spread/flavors.py", line 32, in <module>
    from jelly import setUnjellyableForClass, setUnjellyableForClassTree, setUnjellyableFactoryForClass, unjellyableRegistry
builtins.ImportError: No module named 'jelly'
```

<details><summary>Searchable metadata</summary>

```
trac-id__8621 8621
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8621_rodrigc_spread 8621-rodrigc-spread
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468830102843218 1468830102843218
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/350

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset f1328d4c427648226728b587255c9f2edfeef2f5
```
#!CommitTicketReference repository="" revision="f1328d4c427648226728b587255c9f2edfeef2f5"
Merge pull request #350 from twisted/8621-rodrigc-spread

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8621

In twisted/spread, fix imports on Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
